packages/astro/src/vite-plugin-astro/hmr.ts

```diff
@@ -125,6 +125,7 @@ export async function handleHotUpdate(
 	// TODO: Svelte files should be marked as `isSelfAccepting` but they don't appear to be
 	const isSelfAccepting = mods.every((m) => m.isSelfAccepting || m.url.endsWith('.svelte'));
 	if (isSelfAccepting) {
+		if (/astro\.config\.[cm][jt]s$/.test(file)) return mods;
```

Comment:
@natemoo-re  Oh boy, is this fixing something specific? Not sure what's happening here 🤷

Reply:
This is ignoring the logs that print out `hmr update astro.config.mjs` since they are handled elsewhere